draft-ietf-httpbis-priority.md

```diff
@@ -108,101 +114,100 @@ The term control stream is used to describe the HTTP/2 stream with identifier
 
 
-# Motivation for Replacing HTTP/2 Priorities {#motivation}
+# Motivation for Replacing RFC 7540 Priorities {#motivation}
 
 An important feature of any implementation of a protocol that provides
-multiplexing is the ability to prioritize the sending of information. This was
-an important realization in the design of HTTP/2. Prioritization is a
-difficult problem, so it will always be suboptimal, particularly if one endpoint
-operates in ignorance of the needs of its peer.
-
-HTTP/2 introduced a complex prioritization scheme that uses a combination of
-stream dependencies and weights to describe an unbalanced tree. This scheme has
-suffered from poor deployment and interoperability.
-
-Clients build an HTTP/2 prioritization tree through a series of individual
-stream relationships, which are transferred to the server using HTTP/2 priority
-signals in either of three forms. First, a HEADERS frame with the PRIORITY flag
-set is an explicit signal that includes an Exclusive flag, Stream Dependency
-field, and Weight field. Second, a HEADERS frame with no PRIORITY flag is an
-implicit signal to use the default priority. Third, the PRIORITY frame, which is
-always explicit since it always contains an Exclusive flag, Stream Dependency
-field, and Weight field.
-
-The rich flexibility of tree building is rarely exercised. Experience has shown
-that clients tend to choose a single model optimized for a web use case and
-experiment within the model constraints, or do nothing at all. Furthermore, many
-clients build their prioritization tree in a unique way, which makes it
-difficult for servers to understand their intent and act or intervene
-accordingly.
-
-Many HTTP/2 server implementations do not include support for the priority
-scheme. Some instead favor custom server-driven schemes based on heuristics or
-other hints, such as resource content type or request generation
-order. For example, a server, with knowledge of the document structure, might
-want to prioritize the delivery of images that are critical to user experience
-above other images, but below the CSS files. Since client trees vary, it is
-impossible for the server to determine how such images should be prioritized
-against other responses.
-
-The HTTP/2 scheme allows intermediaries to coalesce multiple client trees into a
+multiplexing is the ability to prioritize the sending of information.
+Prioritization is a difficult problem, so it will always be suboptimal,
+particularly if one endpoint operates in ignorance of the needs of its peer.
+Priority signalling allows endpoints to communicate their own view of priority
+needs, which can be combined with other factors that are considered during the
+peer's prioritization decision making.
+
+RFC 7540 stream priority ({{Section 5.3 of ?RFC7540}}) is a complex system where
+clients signal stream dependencies and weights to describe an unbalanced tree.
+It suffered from poor deployment and interoperability and was deprecated in a
```

Comment:
I would remove this and the next sentence from this part. It seems to break the flow of the text and is already stated in the introduction and repeated later when talking about deprecation and handling of both signals etc.

Reply:
Can you explain more how it breaks the flow? The first two paragraphs in the section introduce the abstract notion of prioritization and priority signals, this paragraph then speaks specifically to RFC 7540 and it's soft deprecation in RFC7540bis. This document needs to provide sufficient context to explain the later hard deprecation, which is different.